template_feed/Microsoft.DotNet.Common.ProjectTemplates.2.1/content/ConsoleApplication-FSharp/Program.fs

```diff
@@ -1,14 +1,13 @@
-﻿// Learn more about F# at http://fsharp.org
+// Learn more about F# at http://fsharp.org
 
 open System
 
 // Define a function to print a message
-// Functions are defined before they are used in F#
-let printMessage message =
-    printfn "%s" message
+let from who =
+    sprintf "from %s" message
```

Comment:
s/message/who/

Also, get to a grammar checker - I strongly suspect it should actually be whom 🤓 

Also, the header comment can be improved on (not by me, but it doesn't really 'print' anything IMO)